presto-hive-function-namespace/src/test/java/com/facebook/presto/hive/functions/TestHiveAggregationFunctions.java

```diff
@@ -112,7 +112,7 @@ public void aggregationFunctions()
     @SuppressWarnings("UnknownLanguage")
     public void check(@Language("SQL") String query, Column... expectedColumns)
     {
-        checkArgument(expectedColumns != null && expectedColumns.length > 0);
+        checkArgument(expectedColumns.length > 0);
```

Comment:
This is incorrect replacement, it will fail when expectedColumns is null

Reply:
Yes, it fails with a null pointer exception, exactly as desired